js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -424,8 +424,8 @@ var TextEditorMask = TextEditorBase.inherit({
         }).bind(this));
     },
 
-    _isBackspaceInputNotHandled: function(e) {
```

Comment:
It is difficult to perceive. I suggest renaming this method to `_backspaceInputHandled`